### PR TITLE
Add project checkin playbook

### PR DESCRIPTION
**Pull request by tim-schilling**

### CODE REVIEW

**Review by ryancheley — APPROVED** [review]
Looks good to me

File: README.md [hunk 1]
```diff
@@ -241,6 +243,24 @@ The expected changes:
 - The repository will be removed from the organization.
 - The repository's teams will be removed from the organization.
 
+## Project Checkin Playbook
+
+1. Review [Django Commons Project Checkins doc][project-checkins-doc]
+2. Look for the project's [GitHub discussion for checkins][project-checkins-discussions], it it doesn't exist create one
```
Comment (williln) [inline README.md:249]:
```suggestion
2. Look for the project's [GitHub discussion for checkins][project-checkins-discussions], if it doesn't exist create one
```

**Review by williln — APPROVED** [review]
One minor grammatical error but looks good! Nothing that should hold up the PR.

**Comment by tim-schilling** [comment]
Thanks team!

**Pull request merged**